routes/utils/errors.js

```diff
@@ -47,5 +47,9 @@ function parse (error) {
     error.message = 'Only admins can access /users'
   }
 
+  if (error.message === 'Document update conflict') {
+    error.message = 'An account with that username already exists'
+  }
+
```

Comment:
Clever! That resolves the issue. But it has a potential side effect: besides user accounts we will also store other documents in the database in future. And as this code is run for all errors, it might change a conflict error for request documents to "an account with that username already exists" which would be confusing.

Instead of fixing it this way, I would suggest we handle this directly in the `accounts.add` method here: https://github.com/hoodiehq/hoodie-server-account/blob/8760b78cb2b0ec3802b05c42c946b642e25f0731/api/accounts/add.js

For comparison, see how we return a custom error if the password is incorrect in `session.add`: https://github.com/hoodiehq/hoodie-server-account/blob/8760b78cb2b0ec3802b05c42c946b642e25f0731/api/sessions/add.js#L25

I’m sorry to make it this complicated :/ if you like, we can merge you fix first, and make a follow up issue for this?